.test-infra/jenkins/job_CloudMLBenchmarkTests_Python.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PhraseTriggeringPostCommitBuilder
+import CronJobBuilder
+
+def cloudMLJob = { scope ->
+  scope.description('Runs the TFT Criteo Examples on the Dataflow runner.')
+
+  // Set common parameters.
+  commonJobProperties.setTopLevelMainJobProperties(scope)
+
+  // Gradle goals for this job.
+  scope.steps {
+    gradle {
+      rootBuildScriptDir(commonJobProperties.checkoutDir)
+      commonJobProperties.setGradleSwitches(delegate, 'master', 360)
```

Comment:
Looks like this is the problematic line, I'd missed this. Should this be:

```
common.setGradleSwitches(delegate)
commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 360)
```

Reply:
Fixed it. Can we run the seed job to confirm?

Reply:
Oh I had a typo, so it failed again:

`ERROR: (job_CloudMLBenchmarkTests_Python.groovy, line 33) No such property: common for class: javaposse.jobdsl.dsl.helpers.step.GradleContext`

https://ci-beam.apache.org/job/beam_SeedJob/10942/console

Instead of `common.setGradleSwitches` it should be `commonJobProperties.setGradleSwitches`